Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -197,6 +207,41 @@ function getTexts(node, path, unixName) {
 	});
 }
 
+function getEntireText(node,path){
+	$('#txtModal').on('show.bs.modal',function(event){
+    	var link = $(event.relatedTarget);
+    	var txtid = link.data('txtid');
+        var title = link.data('txttitle');
+        var author = link.data('txtauthor');
+        var date = link.data('txtdate');
+        var header = "No author and title information provided."
+            if (title != '' || author != '' || date != '') {
+                header = '';
+                if (author != null) {
+                    header += author + ", ";
+                }
+                if (title != null) {
+                    header += "<em>" + title + "</em> ";
+                }
+                if (date != null) {
+                    header += "(" + date + ")";
+                }
+            }            
+            //header += "<br><small>" + txtname + "</small>";
+        $.ajax({ 
+        	type: "GET",
+            url: path + "/public/text/view?conceptUri=" + encodeURIComponent(node.data("conceptUri")) + "&txtid=" + txtid,
```

Comment:
this is still not using information supplied when the modal is opening (needs to look like author or title).